meetup/services/meetup_presenter.py

```diff
@@ -0,0 +1,42 @@
+import time
+from pizzaplace.services.pizza_place_presenter import PizzaPlacePresenter
+from pizzaplace.services.yelp_api import YelpApi
+
+class MeetupPresenter():
+  def __init__(self, meetup, meetup_api, api_response_parser):
+    self.meetup = meetup
+    self.parsed_api_response = self.get_api_response(meetup_api, api_response_parser)
```

Comment:
In Ruby, you might see something like:

```rb
class MeetupPresenter 

  def parsed_api_response
    @parsed_api_response ||= get_api_response(meetup_api, api_response_parser)
  end

end
```

As an idiom for lazy-loading the response. Python doesn't have a built-in `||=` operator, but that's the only magic in the example above. The same thing can be done (a little more verbosely) in Python.